### ISSUE TITLE
VPA deployment script generates bad CRD manifests in modes other than "create"

### ISSUE THREAD

**Issue by aurel333** [kubernetes/autoscaler#5832]
**Which component are you using?**: vertical-pod-autoscaler

Component version: 0.11 but the behavior is the same on master

K8S version: 1.23.10

Expected behavior: The script `vpa-process-yaml.sh` in `print` mode should generate the same resources as in `create` mode, minus the secret.

What happened: The script `vpa-process-yaml.sh` in `print` generated the resources but also generated twice the CustomResourceDefinition manifests from two different files: `vpa-v1-crd-gen.yaml` and `vpa-beta2-crd.yaml`.
`vpa-v1-crd-gen.yaml` is used with the `create` mode so it is expected but `vpa-beta2-crd.yaml` is added in the mix just because the selected mode is `delete`, `diff` or `print`.

How to reproduce it: Run `./hack/vpa-process-yamls.sh print` in the `vertical-pod-autoscaler` directory

**Comment by Shubham82** [comment]
As per the following PR #5345, v1beta 1,2 is deprecated and will remove in the further release (after 3 releases of deprecation).
so maybe that's why it's there or maybe forgot to update it.

**Comment by Shubham82** [comment]
I think we can update the script.
WDYT @jbartosik

**Comment by jbartosik** [comment]
@Shubham82 sounds good to me;

**Comment by Shubham82** [comment]
@jbartosik Removing the following Case statements from the script gives the output only related to v1 (vpa-v1-crd-gen):

```
case ${ACTION} in
delete|diff|print) COMPONENTS+=" vpa-beta2-crd" ;;
esac
```

**Comment by Shubham82** [comment]
I will create a PR for it soon.
/assign

**Comment by Shubham82** [comment]
Fixed this issue here: #5837
